web/src/main/resources/lang/strings.properties

```diff
@@ -153,7 +154,7 @@ page.user.oldPasswordDoesntMatch=Old password doesn't match
 page.showParam.errorNoParamId=No parameter ID supplied
 page.showParam.errorParamNotFound=Can''t find parameter with ID {0}
 
-page.search.asteriskTooltip = The field is case insensitive. \nUse an asterisk as a wildcard at the end of the search string to expand the search results.
+page.search.asteriskTooltip = The field is case insensitive. \nUse an asterisk as a wildcard at the end of the search string to expand the search dataPoints.
```

Comment:
I think it is confusing. At search page, we search test/testExecution not data points